docs/.vuepress/public/presets/toml/gruvbox-rainbow.toml

```diff
@@ -0,0 +1,162 @@
+"$schema" = 'https://starship.rs/config-schema.json'
+
+format = """
+[](orange)\
+$os\
+$username\
+[](bg:yellow fg:orange)\
+$directory\
+[](fg:yellow bg:aqua)\
+$git_branch\
+$git_status\
+[](fg:aqua bg:blue)\
+$c\
+$rust\
+$golang\
+$nodejs\
+$php\
+$java\
+$kotlin\
+$haskell\
+$python\
+[](fg:blue bg:bg3)\
+$docker_context\
+[](fg:bg3 bg:bg1)\
+$time\
+[ ](fg:bg1)\
+$line_break$character"""
+
+palette = 'gruvbox_dark'
+
+[palettes.gruvbox_dark]
+fg0 = '#fbf1c7'
+bg1 = '#3c3836'
+bg3 = '#665c54'
+blue = '#458588'
+aqua = '#689d6a'
+green = '#98971a'
+orange = '#d65d0e'
+purple = '#b16286'
+red = '#cc241d'
+yellow = '#d79921'
```

Comment:
Just a recommendation: as some of those variables are also _Starship_ internal color specifiers (see: [Style Strings](https://starship.rs/advanced-config/#style-strings)), I would recommend using a prefix—like `color_.

Reply:
Prefix added in bf80a4ffc48fd0dc31fcb7f71c19f0b6e6ed86a2.